pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -172,6 +172,19 @@ def pygrb_add_bestnr_cut_opt(parser):
                         "Default 0: all events are considered.")
 
 
+# =============================================================================
+# Wrapper to pick triggers with certain slide_ids 
+# =============================================================================
+def slide_filter(File, data, slide_id=None):
+    """   """
+    mask  = numpy.where(File['network/slide_id'][:]==slide_id)[0]
+
+    if slide_id is not None:
+        return data[mask]
+    else:
+        return data
```

Comment:
Some comments on this function:
* Instead of adding a comment before the function, put the same text as the function's docstring.
* When checking for None, invert the logic: if `slide_id` is None then return `data` right away, otherwise carry on (without an `else` block).
* `File` is a bad choice for a variable name in Python (variables should generally be lowercase, and `file` is a reserved word in Python). Maybe `trigger_file` or `trig_file` are better choices?

Reply:
The docstring is sufficient, anything else just adds lines to the file.